modin/tests/test_utils.py

```diff
@@ -263,10 +265,17 @@ def warns_that_defaulting_to_pandas(prefix=None, suffix=None):
 
     Returns
     -------
-    pytest.recwarn.WarningsChecker
-        A WarningsChecker checking for a UserWarning saying that Modin is
-        defaulting to Pandas.
+    pytest.recwarn.WarningsChecker or contextlib.nullcontext
+        If Modin is not operating in MODIN_NATIVE_DATAFRAME_MODE,a WarningsChecker
+        is returned whic will check for a UserWarning indicating that Modin
+        is defaulting to Pandas. If MODIN_NATIVE_DATAFRAME_MODE is set, a
+        nullcontext is returned to avoid warning about the default to Pandas,
+        as this occurs due  user selecting of MODIN_NATIVE_DATAFRAME_MODE.
```

Comment:
```suggestion
        If Modin is not operating in ``NativeDataframeMode``, a ``WarningsChecker``
        is returned, which will check for a ``UserWarning`` indicating that Modin
        is defaulting to Pandas. If ``NativeDataframeMode`` is set, a
        ``nullcontext`` is returned to avoid the warning about defaulting to Pandas,
        as this occurs due to user setting of ``NativeDataframeMode``.
```